lib/ansible/modules/source_control/gitlab_user.py

```diff
@@ -302,6 +302,9 @@ def main():
     state = module.params['state']
     confirm = module.params['confirm']
 
+    if len(user_password) < 8:
+        module.fail_json(msg="New user's 'password' should contain more than 8 characters.")
```

Comment:
Should this be added to the password description in `DOCUMENTATIOND` block?

Reply:
Yes. I can. I will update.